src/design-tokens/tier-1-definitions/layout.json

```diff
@@ -3,15 +3,15 @@
     "l": {
       "max-width": {
         "value": "71.25rem",
-        "comment": "@deprecated Layout tokens are deprecated and will be removed in a future release"
+        "comment": "@deprecated This should not be used in code or design. It will be removed in a future version of EDS."
```

Comment:
Named all the deprecation comments with the same format, so that we can later find/replace these at the next major